### PR TITLE
Reverts goonchem

### PR DESCRIPTION
**Pull request by ghost**
This took me like 20 minutes. This feature would take nothing but an afternoon to fully revert back to how it was before, no thanks to jordie and his epiccc refactor which ruined the commit history making this hard as fuck to revert.

> goofball tries to shove his boner for goon all over hippiestation
> they hate it
> he shills it here, broken, completely terrible

Good going faggots. You somehow let an even bigger douchebag than me code something big. The worst part is how blissfully ignorant goof is to his own failures. Whatever maintainers thought ever letting goonchem pass is an absolute moron, through and through.

Not only do I want goofball banned from this respository, but for tkdrg to lose maintainer and allura to be banned as well. Goon shills need to fucking go and leave our code alone, thanks.

### CODE REVIEW

**Comment by Thunder12345** [comment]
:fu::droplet::baby::droplet::gun::arrow_right::skull:

**Comment by Deantwo** [comment]
I am against this. It is working and we are through the worst of it.
@Iamgoofball also has a number of PR fixes right now that are waiting.

The only thing that I still don't like in new chem is the heat thing. #7250
I still think that the temperature should only be in the ChemHeater. Once you take the beaker out it should not be warm anymore, this would mean that creating chemicals that require heat actually become something that takes time because the heater has to reheat it every time.

**Comment by paprka** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2597
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2555

**Comment by Razharas** [comment]
I heard this much salt can give you kidney stones
Also i like the idea of patches

**Comment by Chocobro** [comment]
:-1: goonchem is being worked out. wait for those fixes to be live and player-reviewed before reverting it outright, please

**Comment by paprka** [comment]
Except patches are worthless bad gameplay and everyone knows this

**Comment by Thunder12345** [comment]
That poll happened before a lot of the bugs got fixed though, you'll probably get a different result now there's been time to get over the initial mad and get bugfixes through.

**Comment by Deantwo** [comment]
>https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2597
>https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2555

Those are half a month old. At that time it WAS broken.
As of right now it is working. Do a new poll.

**Comment by paprka** [comment]
>add shitty feature that's broken as hell
>patch it over the course of several months
>can't revert it anymore

oh well time to stick with the feature ;)

goofball literally fucking admitted to this being his intention, how is this okay?

**Comment by Firecage** [comment]
I also enjoy Goonchem(Goofchem) MUCH more than I enjoyed Oldchem.

**Comment by Steelpoint** [comment]
I fail to see any significant difference between Star Trek Chem and Goonchem outside of a significantly larger amount of harmful chemicals that can be made with far less effort than good healing chemicals.

I continue to maintain that a Chemistry overhaul MUST be done from the ground up.

**Comment by paprka** [comment]
Let's make another epic forum poll, or better yet, a server poll, to decide then. See what the players think! After all, this is for the players! :santa:

**Comment by Steelpoint** [comment]
I think a server poll is the best way forward.

Before anyone says, I don't think we care if Goonchem is still being worked on, it should have been ready to go and functional when it was merged, not having it still fuck shit up a month later.

**Comment by paprka** [comment]
I'll wait for another landslide poll then

**Comment by Deantwo** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2802
So people can vote.

**Comment by AnonymousNow** [comment]
"A landslide revert poll? Well, let's ignore that and just change a bunch of things. See? That poll was for a version we had two weeks ago, we can ignore it!"

I'm disgusted. It's evident that coderbus does not listen to polls.

**Comment by Miauw62** [comment]
honestly, i agree with this revert.
nobody likes goonchem, even after like a month.
there's fucktons of weird bugs, etc, etc.

tg needs a serious medical rewrite, not fifty separate overhauls that dont touch eachother.

**Comment by Miauw62** [comment]
oh well we'll wait for another poll.

**Comment by Steelpoint** [comment]
No. I think Goonchem was a mistake.

We need to do it from the ground up, Goonchem is nothing more than a hamfisted attempt to improve our chemical system, except it does almost nothing except give antags more ways to kill people in stupid ways.

**Comment by Jordie0608** [comment]
At the time of commenting a bunch of the code with spawning bottles etc. with reagents will bork in this pr.

Since it's my fault for doing a refactor in the middle of the goonchem kerfuffle, if you don't want to fiddle around and fix them then I'll do a re-refactor if this is merged.

**Comment by dannno** [comment]
The goonchem merge was awful. Hardly tested, reviewed by one person IIRC. That is simply not enough for a change this big. The impact it had was more than anticipated, in a very bad way. 
Regardless of whether it would benefit /tg/ to get goonchem in the future, for now this should be reverted. Maybe next time it can be actual looked over before being implemented so it doesn't fuck a bunch of shit up as bad.

**Comment by Deantwo** [comment]
Yeah I agree @dannno, but I still don't thing reverting is the correct action at this point.

**Comment by Deantwo** [comment]
Can we get a poll ingame too?
And for crying out loud, be specific, what are we talking about reverting?
ChemHeaters? Chem name changes? Patches? New chems? Old chems removal?

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -194,12 +194,12 @@
 	item_state = "r_suit"
 	item_color = "red_suit"
 
-/obj/item/clothing/under/suit_jacket/charcoal
-	name = "charcoal suit"
-	desc = "A charcoal suit and red tie. Very professional."
-	icon_state = "charcoal_suit"
-	item_state = "charcoal_suit"
-	item_color = "charcoal_suit"
+/obj/item/clothing/under/suit_jacket/anti_toxin
```
Comment (Deantwo) [inline code/modules/clothing/under/miscellaneous.dm]: What.

Comment (dannno) [inline code/modules/clothing/under/miscellaneous.dm]: pfffhahahahahaha

**Comment by Iamgoofball** [comment]
the "string it out over a set of prs to make it hard to revert" was a joke jesus christ

i'm looking into plans to make trek chems come back, calm your shit

**Comment by Deantwo** [comment]
Created a new thread with a poll that is less "revert, don't revert" focused.
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2806

**Comment by optimumtact** [comment]
13:20 <@hg|work> im not going to revert goonchem

Can you please focus on improvements now instead of wasting time here. Paprka you need to move on.

**Comment by dannno** [comment]
>paprka you need to move on
>paprka

yeah not like we did a bunch of votes that indicated a mass dislike of goonchem from a majority of people or anything
it's obviously just paprka's grudge

**Comment by optimumtact** [comment]
Sure thing

**Comment by Deantwo** [comment]
Yeah but reverting it all is the wrong way to fix it.
We need to find out why people are so unhappy with it and makr it better.

But my guess is that people are mostly against it because they have to relearn the system.
So lets see what [my new poll](https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2806) says.

**Pull request closed without merging**